ui/desktop/app/routes/scopes/scope/projects/targets/target.js

```diff
@@ -17,6 +18,10 @@ export default class ScopesScopeProjectsTargetsTargetRoute extends Route {
   @service router;
   @service session;
 
+  // =attributes
+
+  @tracked connectionError = false;
```

Comment:
Do we need this to be tracked?

Should we also call it `isConnectionError` since it's a boolean?